app/main.py

```diff
@@ -1,7 +1,23 @@
+import requests
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+
+    API_KEY = "b47adef0ac6d4d919b5131859253105"
```

Comment:
You must not store API keys or any confidential data in a public GitHub repository.